contracts/StreamManager.sol

```diff
@@ -241,8 +212,12 @@ contract StreamManager is IStreamManager, ReentrancyGuard {
      */
     function terminate(address _payee) external onlyPayer notTerminated {
         uint256 terminatedAt = block.timestamp;
+        OpenStream storage streamInstance = streamInstances[_payee];
         if (!isPayee[_payee]) revert NotPayee();
-        if (streamInstances[_payee].terminatedAt != 0) revert AlreadyTerminatedOrTerminating();
+        if (streamInstance.terminatedAt != 0) revert AlreadyTerminatedOrTerminating();
+        /// Terminate in cliff period
+        if (streamInstance.createdAt + streamInstance.cliffPeriod >= block.timestamp)
+            isPayee[_payee] = false;
         streamInstances[_payee].isTerminated = true;
         streamInstances[_payee].terminatedAt = terminatedAt;
```

Comment:
@accompany1205 you already have the reference of the instance, just update it accordingly

Reply:
Fixed